execution/commitment/hex_patricia_hashed.go

```diff
@@ -911,18 +911,9 @@ func (hph *HexPatriciaHashed) computeCellHash(cell *cell, depth int16, buf []byt
 			hashedKeyOffset = depth - 64
 		}
 		singleton := depth <= 64
-		koffset := hph.accountKeyLen
```

Comment:
this needed for correct hashing of storage keys

Reply:
let the CI end first :), the tests pass